src/jetson/video_capturer.py

```diff
@@ -54,9 +56,29 @@ def update(self):
         """Get next frame in video stream"""
         while self.running:
             if self.capture.isOpened():
-                _, self.frame = self.capture.read()
+                ret, self.frame = self.capture.read()
+                if not ret:
+                    self.running = False
+                    
             time.sleep(.01)
 
+    def reboot(self):
+        ret = False 
```

Comment:
what is this for?

Reply:
docstring I added was not pushed in commit, it should be there now.